src/test/libradosstriper/io.cc

```diff
@@ -52,6 +52,17 @@ TEST_F(StriperTest, Stat) {
   ASSERT_EQ(-ENOENT, rados_striper_stat(striper, "nonexistent", &psize, &pmtime));
 }
 
+TEST_F(StriperTest, Stat2) {
+  uint64_t psize;
```

Comment:
i'd suggest always initialize the variable with a "safe" value.

Reply:
the "p" in the variable name could be misleading. might want to use `size` instead.